python/mxnet/contrib/onnx/mx2onnx/export_onnx.py

```diff
@@ -118,31 +118,39 @@ def forward_pass(inputs, sym, arg_params, aux_params, output_label):
                       if graph_input not in arg_params and graph_input not in aux_params
                       and graph_input != output_label]
 
+        data_forward = []
         data_shapes = []
         # Adding extra dimension of batch_size 1 if the batch_size is different for multiple inputs.
         for idx, input_name in enumerate(data_names):
-            data_shapes.append((input_name, inputs[idx].shape))
+            val = inputs[idx]
+            data_shapes.append((input_name, val.shape))
+            data_forward.append(nd.array(val))
 
         # create module, passing cpu context
         ctx = context.cpu()
-        test_mod = mod.Module(symbol=sym, data_names=data_names, context=ctx, label_names=None)
-        test_mod.bind(for_training=False, data_shapes=data_shapes, label_shapes=None)
 
-        # initializing parameters for calculating result of each individual node
-        if arg_params is None and aux_params is None:
-            test_mod.init_params()
-        else:
-            test_mod.set_params(arg_params=arg_params, aux_params=aux_params, allow_missing=True)
+        # module bind method requires all data to have same batch size,
```

Comment:
why not use sym.bind whatsoever

Reply:
tests for batchnorm and conv2d alone fail due to missing keys. will debug further and update.

Reply:
@zhreshold  Will debug this and submit a separate PR for this commit and InstanceNorm (which depends on this). This will make sure that HardSigmoid and the comparison operators can be reviewed/merged instead of waiting on this. Hope that's ok.

Reply:
@vandanavk  making sure only high quality code get merged into master is critical. Having pushing feature request isn't a good reason to bypass this restriction.

Reply:
Agreed @zhreshold. Opened a WIP PR https://github.com/apache/incubator-mxnet/pull/12920 to continue debug. 

The changes in this PR on HardSigmoid and comparison operators are not impacted by the changes in PR https://github.com/apache/incubator-mxnet/pull/12920.